Formula/libgraphqlparser.rb

```diff
@@ -14,6 +14,8 @@ class Libgraphqlparser < Formula
     sha256 cellar: :any, high_sierra:   "64779ec3108d9eef789d279abfafa90437c6a76b2ed3973d45979cd1051dc170"
   end
 
+  deprecate! date: "2020-04-20", because: "requires Python 2 to build"
```

Comment:
Some dates I was considering:
- `2020-04-20` was final Python 2 (2.7.18) release date.
- `2020-01-01` was official Python 2 EOL date.

Reply:
Let's just go with today, so that it's easier for us to figure out if enough time has passed to `disable!` it.